docs/windows-design.md

```diff
@@ -0,0 +1,249 @@
+# Running Antrea on Windows
+
+Antrea supports running on Windows worker Node. In this case, the Windows Node should
+be a worker in the cluster, while the master Node must be Linux. On Windows Node, 
+Antrea sets up an overlay network to forward packets between Nodes and implements
+NetworkPolicies. 
+
+## Design
+
+On Windows, the Host Networking Service (HNS) is a necessary component to support container
+networking. For Antrea on Windows, “Transparent” mode is chosen for the HNS network. In this
+mode, containers will be directly connected to the physical network through an **external** Hyper-V switch. 
+
+OVS is working as the extension of the external Hyper-V switch, which is bound with the HNS 
```

Comment:
is "bound" the best word here? I thought the Hyper-V switch was *created* by HNS

I would suggest the following but let me know if I misunderstood: "OVS is working as a forwarding extension for the external Hyper-V switch which was created by HNS"